doctr/models/predictor/base.py

```diff
@@ -79,7 +79,10 @@ def _process_predictions(
             boxes, angles = zip(*loc_preds)
             # Rotate back boxes if necessary
             if allow_rotated_boxes:
-                boxes = [rotate_boxes(page_boxes, angle) for page_boxes, angle in zip(boxes, angles)]
+                boxes = [rotate_boxes(page_boxes[:, :-1], angle) for page_boxes, angle in zip(boxes, angles)]
+            # if straight boxes, convert to bbox
+            else:
+                boxes = [page_boxes[:, :-1] for page_boxes in boxes]
```

Comment:
sorry about this, but since #641, I think this needs to be updated :sweat_smile:

Reply:
merging done ..